examples/workflows/main.go

```diff
@@ -16,9 +16,9 @@ limitations under the License.
 
 // Example Allow List based access plugin.
```

Comment:
This (as well as the readme) still mentions an allow list, but you've removed all traces of the allow list.

Documentation that doesn't match the code creates additional support burden.

What is your intent here? That the plugin will attempt to automatically approve every single access request it sees?

Reply:
The allow list would attempt to approve all requests for users in the allow list regardless if the plugin role had permission or not. Removing the allow list so the plugin now will only approve requests for roles the plugin is authorized to review based on RBAC.

Will go over and remove all traces of allow list.

Reply:
I guess the concern is you're basically removing any notion of telling the plugin what to approve, and just telling to to try to approve everything it can, and letting RBAC catch errors. 

Will this clutter the audit log with access denied errors?